core/templates/dev/head/pages/exploration_player/PlaythroughService.js

```diff
@@ -49,6 +49,16 @@ oppia.factory('PlaythroughService', [
     var cycleIdentifier = {};
     var visitedStates = [];
 
+    var extractOldQuitAction = function() {
```

Comment:
Add JSDoc explaining why this needs to be done.

Reply:
Done.